src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -81,10 +81,6 @@ class YamlFileLoader extends FileLoader
         'shared' => 'shared',
         'lazy' => 'lazy',
         'public' => 'public',
-        'abstract' => 'abstract',
-        'deprecated' => 'deprecated',
-        'factory' => 'factory',
-        'arguments' => 'arguments',
```

Comment:
Let's discuss more the use of `factory` inside defaults and/or instanceof sections in #22189.